internal/venice-common/src/main/java/com/linkedin/venice/status/PushJobDetailsStatus.java

```diff
@@ -6,7 +6,7 @@
 
 
 public enum PushJobDetailsStatus {
-  STARTED(0), COMPLETED(1), ERROR(2), NOT_CREATED(3), UNKNOWN(4), TOPIC_CREATED(5), WRITE_COMPLETED(6), KILLED(7),
+  STARTED(0), COMPLETED(1), ERROR(2), NOT_CREATED(3), UNKNOWN(4), TOPIC_CREATED(5), DATA_WRITER_COMPLETED(6), KILLED(7),
```

Comment:
Do we need to change this name? I am afraid it might lead to any compatibility issue.

Reply:
I don't think it will be a problem since we don't use these names anywhere - just their valus